x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/prepackaged_rules/403_response_to_a_post.json

```diff
@@ -1,24 +1,24 @@
-{
-  "description": "A POST request to web application returned a 403 response, which indicates the web application declined to process the request because the action requested was not allowed",
-  "false_positives": [
-    "Security scans and tests may result in these errors. Misconfigured or buggy applications may produce large numbers of these errors. If the source is unexpected, the user unauthorized, or the request unusual, these may indicate suspicious or malicious activity."
-  ],
-  "index": [
-    "apm-*-transaction*"
-  ],
-  "language": "kuery",
-  "name": "Web Application Suspicious Activity: POST Request Declined",
-  "query": "http.response.status_code:403 and http.request.method:post",
-  "references": [
-    "https://en.wikipedia.org/wiki/HTTP_403"
-  ],
-  "risk_score": 47,
-  "rule_id": "a87a4e42-1d82-44bd-b0bf-d9b7f91fb89e",
-  "severity": "medium",
-  "tags": [
-    "APM",
-    "Elastic"
-  ],
-  "type": "query",
-  "version": 2
+{

```

Comment:
line endings are screwed up again.
should I fix this like the other one?

Reply:
Let's let this test run complete first, I'd like to see the test result. After that yes, please do fix later today or tomorrow. I will hold off on merging until we're happy.

Reply:
It would be great if this could be fixed upstream. @randomuserid where does that toml -> json code live?

Reply:
Just ensure that it is from a package generated from https://github.com/elastic/siem-rules/pull/137 with the hash 
```
- sha256: 65a4b77aaba364acb7196c4d0ffc8c0dd9476f63bb96f9c5105e59ddc01a8f3d
```

Reply:
Oh, nice! Thanks @brokensound77 and @rw-access for being on top of this.